docker-compose.yml

```diff
@@ -3,17 +3,18 @@ services:
   db:
     image: mysql:5.7
     environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: '1'
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+      MYSQL_DATABASE: ${TRACKS_DB:-tracks}
     volumes:
-      - "db-data:/var/lib/mysql"
+      - db-data:/var/lib/mysql
   web:
     build: .
+    command: rails s
```

Comment:
This is unnecessary, IMO, as I've modified the binstubs to use the docker environment by default, so that if you follow standard rails practices and run `/bin/rails s`, it'll start the containers for you.

Reply:
I tried that, I ended up adding it as a command because it's not working for me for whatever reason. Let me try some more to figure out what is not working for me. Anyway I agree with your point, I'll get rid of it

Reply:
Interesting. I don't want you to struggle to get it working with the binstubs in the repo, so if it's not a quick fix, I'd rather just leave it in.

Reply:
@mattr- I think I've found it! Isn't that the `./script/server` meant to be run to start the server and then the `./bin/rails *` as a way to run the console or other scripts that might otherwise conflict with the already open port?

If that is the case I might simply need to change the documentation, right?

Reply:
Oh! Yes! `script/server` is definitely meant to be run to start the server so that we expose the service ports to the outside world. If you could update the documentation, that would be great!